test/lit/print_elements.test

```diff
@@ -7,7 +7,7 @@ RUN: %lldb -b -o 'command script import print_elements' %t | FileCheck %s
 Check target.max-children-count values are correctly translated to "print
 elements" values.
 
-CHECK: gdb.parameter('print elements'): None
+CHECK: gdb.parameter('print elements'): 256
```

Comment:
The behavior where a negative number means "no limit" was recently removed by https://github.com/llvm/llvm-project/pull/105460. I think it would be better to just remove this CHECK, I don't see a need to check for a specific default value and it makes the test brittle in case lldb decides to change the default.

If you want, you could also simplify the `parameter` function around line 950 to
```
def parameter(s: str) -> Any:
    # gdb's 'print elements' is used for number of array elements to print and
    # also max number of chars in a string. lldb has 'target.max-children-count'
    # and 'target.max-string-summary-length', but max-children-count seems like
    # a closer match.
    if s == "print elements":
        return int(_GetSetting('target.max-children-count'))
    return None
```
but we can also do that as a follow-up if you prefer.